pkg/kubelet/dockershim/libdocker/kube_docker_client.go

```diff
@@ -522,6 +536,28 @@ func (d *kubeDockerClient) ResizeContainerTTY(id string, height, width uint) err
 	})
 }
 
+func (d *kubeDockerClient) GetContainerStats(id string) (*dockertypes.StatsJSON, error) {
```

Comment:
nit: Add a comment that this is only used on windows.

Reply:
Fixed.